### PR TITLE
chore: add xlx file types to fields with xlsx

### PR DESCRIPTION
**Pull request by marcellmueller**

### CODE REVIEW

File: app/formSchema/uiSchema.ts [hunk 1]
```diff
@@ -603,7 +603,7 @@ const uiSchema = {
     'ui:options': {
       maxLength: MAX_LONG_INPUT_LENGTH,
       label: false,
-      fileTypes: '.xlsx',
+      fileTypes: 'xls, .xlsx',
```
Comment (matthieu-foucault) [inline app/formSchema/uiSchema.ts:606]:
```suggestion
      fileTypes: '.xls, .xlsx',
```
Should there be a period in front of each of those? 
Additionally, can we create an `EXCEL_FILE_EXTENSIONS` constant in this file to ensure consistency?

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/formSchema/uiSchema.ts [hunk 2]
```diff
@@ -549,7 +551,7 @@ const uiSchema = {
     'ui:options': {
       allowMultipleFiles: true,
       maxLength: MAX_LONG_INPUT_LENGTH,
-      fileTypes: '.xlsx, .mpp',
+      fileTypes: '.xls, .xlsx, .mpp',
```
Comment (matthieu-foucault) [inline app/formSchema/uiSchema.ts:554]:
```suggestion
      fileTypes: `${EXCEL_FILE_EXTENSIONS}, .mpp`,
```

File: app/formSchema/uiSchema.ts [hunk 3]
```diff
@@ -575,7 +577,7 @@ const uiSchema = {
     'ui:options': {
       maxLength: MAX_LONG_INPUT_LENGTH,
       fileTypes:
-        '.eml, .msg, .txt, .pdf, .doc, .docx, .xlsx, .xlsx, .xml, .jpg, .jpeg, .png',
+        '.eml, .msg, .txt, .pdf, .doc, .docx, .xls, .xlsx, .xml, .jpg, .jpeg, .png',
```
Comment (matthieu-foucault) [inline app/formSchema/uiSchema.ts:580]:
```suggestion
        `.eml, .msg, .txt, .pdf, .doc, .docx, .xml, .jpg, .jpeg, .png, ${EXCEL_FILE_EXTENSIONS}`,
```

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/formSchema/uiSchema.ts (hunk 3 above)
Comment (marcellmueller) [reply app/formSchema/uiSchema.ts:580]: I dunno I considered that but it seems messy and less readable.

Comment (matthieu-foucault) [reply app/formSchema/uiSchema.ts:580]: It's a bit more verbose but it ensures that you're consistent. Ideally we'd have all of these extensions in named constants and we'd just join all of them. Some of the extensions listed in this file are not necessarily well known (e.g. `.mpp`) and should really be in a constant.

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**